cdhweb/static_src/global/components/home-hero.scss

```diff
@@ -90,38 +94,12 @@
   );
 
   @include md {
-    --box-3d-side-size: 56px;
-  }
-
-  &::before,
-  &::after {
-    content: '';
-    position: absolute;
+    --three-dee-side-size: 56px;
```

Comment:
you've still got `--box-3d-side-size` on line 82, I don't think that should be there

Reply:
Good spotting